### PR TITLE
Remove net_wealth from PSID, update data folder, and code formatting in psid_data_setup.py

### PR DESCRIPTION
**Pull request by MaxGhenis**
I messed up #15 after getting behind master and accidentally adding a big file, so trying again. This PR:
* Removes `net_wealth`, which isn't in `.RData` (see #21)
* Fixes the path to `.RData`
* Improves code formatting with `black`

The only substantive changes are in line 25 and 148; everything else is code formatting (sorry these aren't in separate commits).

### CODE REVIEW

**Comment by MaxGhenis** [comment]
I also verified that the script runs. It produced the following files in `ogusa_calibrate`:
```
first_stage_reg_results.pkl
hh_id_two_statuses.csv
psid_lifetime_income.pkl
psid_to_check.csv
```
I added `ogusa_calibration/*.pkl` and `ogusa_calibration/*.csv` to the `.gitignore`, though we probably want to move these files elsewhere, or potentially include them. Only `psid_lifetime_income.pkl` exceeds GitHub's 100MB limit (130MB), and some other file types might get that below the limit.

**Comment by MaxGhenis** [comment]
@jdebacker please let me know if you'd like more info on this to decide on merging.

File: ogusa_calibrate/psid_data_setup.py [hunk 1]
```diff
@@ -7,22 +7,23 @@
...
+"""
 This script takes PSID data created from psid_download.R and:
 1) Creates variables at the "tax filing unit" (equal to family unit in
    PSID since there is no info on the filing status chosen).
 2) Selects a sample of observations to work with (e.g., dropping very
    old, very low income, etc.).
 3) Computes a measure of lifetime income and places each household into
    a lifetime income percentile group
-'''
+"""
```
Comment (jdebacker) [inline ogusa_calibrate/psid_data_setup.py:22]: Are the double quotes everywhere a `black` thing @MaxGhenis?  You're killing me with these :-)

**Comment by jdebacker** [comment]
@MaxGhenis This is looking good to me.  I especially like including those files in the git ignore.  I don't think they need to be checked into the repo.

File: ogusa_calibrate/psid_data_setup.py (hunk 1 above)
Comment (MaxGhenis) [reply ogusa_calibrate/psid_data_setup.py:22]: Yep will definitely make these separate commits in the future!

**Comment by rickecon** [comment]
@MaxGhenis @jdebacker . I think this PR is ready to merge. But I don't understand the "conflicts that must be resolved" above. When I click on `Resolve conflicts`, it shows me a version of the `psid_data_setup.py` file that is not consistent with the changes shown in the "Files changed" tab. Maybe this is just a case where Max's branch is behind the current state of the repo, and he needs to do a git fetch/merge/resolve conflicts on his end. Any clarification here would be helpful.

**Comment by jdebacker** [comment]
@MaxGhenis When conflicts are resolved on this branch, I'm very happy to merged this.  B/c wealth variables are important and the `psidR` package no longer can be used for them, I may check back into this repo the old PSID data with wealth, just in case we need it.  But I like the updates here so that things work with the latest `psidR` package.

**Pull request closed without merging**